makeown.lua

```diff
@@ -280,7 +280,7 @@ local function fix_histfig(unit)
             ::continue::
         end
         -- If you're makeown-ing an enemy of your civilization or group, people will feel vengeful without this
-        if df.histfig_entity_link_enemyst:is_instance(el) then
+        if el and df.histfig_entity_link_enemyst:is_instance(el) then
```

Comment:
I mean `el` can be a stale reference. the call to `entity_link` above can delete the memory that `el` points to.

Reply:
You need to reorganize this loop such that  you guarantee nothing touches `el` after a call to `entity_link`.

The API of `entity_link` doesn't make it clear at all that there is a possible side effect of deleting the memory that `el` points to, so a "proper" fix would be to rearchitect this section of the code, but for the purposes of this PR, just not crashing in the case where `entity_link` deletes `el` would be enough.